extension/js/common/ui/key-import-ui.ts

```diff
@@ -59,7 +60,7 @@ export class KeyImportUi {
   }
   public onBadPassphrase: VoidCallback = () => undefined;
 
-  public initPrvImportSrcForm = (acctEmail: string, parentTabId: string | undefined) => {
+  public initPrvImportSrcForm = (acctEmail: string, parentTabId: string | undefined, view?: SetupView) => {
```

Comment:
It's a wrong decision to add dependency to `SetupView` in this "common" class. Passing an array of `submitKeyForAddrs`  should be enough